src/components/webcams/WebcamGrid.vue

```diff
@@ -49,6 +53,7 @@ import { GuiWebcamStateWebcam } from '@/store/gui/webcams/types'
         'webcam-ipstreamer': Ipstreamer,
         'webcam-hlsstreamer': Hlsstreamer,
         'webcam-webrtc-camerastreamer': WebrtcCameraStreamer,
+        'webcam-janus': JanusStreamer,
```

Comment:
pls rename it to webcam-webrtc-janus (like the webrtc-camerastreamer example one line above)